app/controllers/events_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
```

Comment:
EventsController should be in `api/v1` folder

Reply:
Put controller to `api/v1` folder and added `module Api` and `module V1`